Makefile

```diff
@@ -91,12 +87,15 @@ $(BUILD_DIR)/linux-amd64/crc: $(SOURCES)
 $(BUILD_DIR)/windows-amd64/crc.exe: $(SOURCES)
 	GOARCH=amd64 GOOS=windows go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/windows-amd64/crc.exe ./cmd/crc
 
+build-aux/crc-embedder: $(SOURCES) build-aux/crc-embedder.go
+	go build --tags="build" -ldflags="$(LDFLAGS)" -o build-aux/crc-embedder ./build-aux
```

Comment:
Shouldn't we use `BUILD_DIR` for all the binary generation part? Following patch have all the diff which I think should be considered.

```patch
-build-aux/crc-embedder: $(SOURCES) build-aux/crc-embedder.go
-       go build --tags="build" -ldflags="$(LDFLAGS)" -o build-aux/crc-embedder ./build-aux
+build-aux/crc-embedder: $(SOURCES)
+       go build --tags="build" -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/crc-embedder ./build-aux
 
 .PHONY: cross ## Cross compiles all binaries
 cross: $(BUILD_DIR)/macos-amd64/crc $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/windows-amd64/crc.exe
@@ -200,6 +200,6 @@ check_bundledir:
 
 embed_bundle: LDFLAGS += $(BUNDLE_EMBEDDED)
 embed_bundle: clean cross build-aux/crc-embedder check_bundledir $(HYPERKIT_BUNDLENAME) $(HYPERV_BUNDLENAME) $(LIBVIRT_BUNDLENAME)
-       build-aux/crc-embedder embed --goos=darwin --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/macos-amd64/crc
-       build-aux/crc-embedder embed --goos=linux --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/linux-amd64/crc
-       build-aux/crc-embedder embed --goos=windows --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/windows-amd64/crc.exe
+       $(BUILD_DIR)/crc-embedder embed --goos=darwin --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/macos-amd64/crc --log-level debug
+       $(BUILD_DIR)/crc-embedder embed --goos=linux --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/linux-amd64/crc --log-level debug
+       $(BUILD_DIR)/crc-embedder embed --goos=windows --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/windows-amd64/crc.exe --log-level debug
```

Reply:
dunno what are the conventions for go + Makefile :) if you prefer things this way, no objections from me, I'll change.